mongo/integration/unified/operation.go

```diff
@@ -15,12 +15,13 @@ import (
 )
 
 type operation struct {
-	Name           string         `bson:"name"`
-	Object         string         `bson:"object"`
-	Arguments      bson.Raw       `bson:"arguments"`
-	ExpectedError  *expectedError `bson:"expectError"`
-	ExpectedResult *bson.RawValue `bson:"expectResult"`
-	ResultEntityID *string        `bson:"saveResultAsEntity"`
+	Name                 string         `bson:"name"`
+	Object               string         `bson:"object"`
+	Arguments            bson.Raw       `bson:"arguments"`
+	IgnoreResultAndError bool           `bson:"ignoreResultAndError"`
```

Comment:
Adding this field was the only change to this type. The rest of the changes are due to struct alignment in gofmt.